Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -400,6 +411,91 @@ public void TestUserProfileStaysOutOfFingerprints()
             XAssert.IsTrue(json.Contains(Path.GetFileName(Environment.GetFolderPath(Environment.SpecialFolder.InternetCache)).ToUpperInvariant()));
         }
 
+        [Fact]
+        public void TestModifiedExecutableWhenUntrackedStaysOutOfFingerprints()
+        {
+            var exectuablePath = "/x/pkgs/tool.exe";

```

Comment:
1. typo: `exectuable` -> `executable`
2. call `X()` here once instead of calling it everywhere else where this path is used.
```suggestion
            var executablePath = X("/x/pkgs/tool.exe");
```